src/third_party/mozjs-60/platform/ppc64le/freebsd/build/Unified_cpp_js_src0.cpp

```diff
@@ -0,0 +1,55 @@
+#define MOZ_UNIFIED_BUILD
```

Comment:
Let's remove the all generated sources from this review. I think it is better if you hold them on your side, since we have no way to regenerate this when we need to.